app/src/main/java/pm/gnosis/heimdall/ui/onboarding/password/PasswordConfirmActivity.kt

```diff
@@ -47,54 +41,74 @@ class PasswordConfirmActivity : SecuredBaseActivity() {
         layout_password_confirm_password.setOnEditorActionListener { _, actionId, _ ->
```

Comment:
We could use the rxBinding extension and merge the stream with the button clicks stream